packages/scandipwa/src/component/MyAccountSignIn/MyAccountSignIn.component.js

```diff
@@ -95,6 +98,7 @@ export class MyAccountSignIn extends PureComponent {
                 <div block="MyAccountOverlay" elem="SignInButton">
                     <button block="Button">{ __('Sign in') }</button>
                 </div>
+                { isLoading && <Loader /> }
```

Comment:
You can just pass isLoading props into Loader, so it will show loader only when true.

Reply:
Have changed!